src/static/js/libs/gamepad.js

```diff
@@ -562,12 +562,12 @@
     if (this.gamepads.length > 0) {
       this._fire(Gamepad.Event.TICK, this.gamepads);
     }
-    //window.requestAnimationFrame(function() {
-    //	self._update();
-    //});
-    setTimeout(function () {
-      self._update();
-    }, 100);
+    window.requestAnimationFrame(function() {
```

Comment:
Just making sure we checked performance when re-enabling this. It generates more javascript processing gamepad input at 60Hz vs 10hz and won't make any difference if we have the throttle that is sending the position updates still at 10hz.  Things to check.